diff --git a/node_modules/vite/client.d.ts b/node_modules/vite/client.d.ts
index 2586f07..f3fe7c4 100644
--- a/node_modules/vite/client.d.ts
+++ b/node_modules/vite/client.d.ts
@@ -84,7 +84,7 @@ declare module '*.gif' {
   export default src
 }
 declare module '*.svg' {
-  const src: string
+  const src: React.ComponentType<React.SVGProps<SVGSVGElement>>
   export default src
 }
 declare module '*.ico' {
